proto/frequenz/api/common/v1/components/grid.proto

```diff
@@ -0,0 +1,26 @@
+// Frequenz definitions for grid connection points.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.components.grid;
+
+// A representation of a grid connection point.
+message Grid {
```

Comment:
How about renaming this to `GridConnection` or `GridFuse` instead of just Grid? Grid isn't really a "component".

Reply:
ChatGPT suggests that the most widely known name for this point is `Point of Common Coupling (PCC)`. However, I prefer the suggestion `Grid Interconnection Point`. What do you suggest?

![image](https://github.com/frequenz-floss/frequenz-api-common/assets/63402123/bacc088f-1331-4699-b56c-e22a89155eca)

Reply:
Or just `GridConnectionPoint` maybe?

Reply:
https://en.wikipedia.org/wiki/Microgrid#Point_of_common_coupling_(PCC)

It looks like ChatGPT is right if you look at Wikipedia. Of course GridConnectionPoint is a more general term describing the same concept without using the more sophisticated terminology. Which way we ever go we can of course also use the documentation to clarify. Why don't you put out a vote in Rnd or something?

Reply:
As discussed in slack, changed it to `GridConnectiopnPoint`. The idea is to be more approachable to users that do not have an EE background.

Reply:
Yes, however don't forget to mention PCC in the documentation and briefly explain why we choose grid connection point (GCP) over PCC.